examples/run_onepulse_nutation.py

```diff
@@ -0,0 +1,91 @@
+"""
+One Pulse Experiment
+====================
+
+Not an echo detection but rather just capturing
+the FID with a 4-step phase cycle. 
+"""
+
+from pyspecdata import getDATADIR
+from numpy import r_, linspace
+import sys
+import os
+import SpinCore_pp
+import SpinCore_pp as spc
+from SpinCore_pp.ppg import generic
+from datetime import datetime
+
+my_exp_type = "ODNP_NMR_comp/nutation"
+assert os.path.exists(getDATADIR(exp_type=my_exp_type))
+desired_p90_range_us = linspace(0.5, 15, 20, endpoint=False)
+# {{{importing acquisition parameters
+config_dict = SpinCore_pp.configuration("active.ini")
+(
+    nPoints,
+    config_dict["SW_kHz"],
+    config_dict["acq_time_ms"],
+) = spc.get_integer_sampling_intervals(
+    SW_kHz=config_dict["SW_kHz"],
+    time_per_segment_ms=config_dict["acq_time_ms"],
+)
+# }}}
+# {{{add file saving parameters to config dict
+config_dict["type"] = "FID_nutation"
+config_dict["date"] = datetime.now().strftime("%y%m%d")
+config_dict["FID_nutation_counter"] += 1
+# }}}
+# {{{ command-line option to leave the field untouched (if you set it once, why set it again)
+spc.set_field(sys.argv, config_dict)
+# }}}
+# {{{set phase cycling
+ph1_cyc = r_[0, 1, 2, 3]
+nPhaseSteps = 4
+# }}}
+prog_p90_us = spc.prog_plen(desired_p90_range_us)
```

Comment:
This function now takes beta and amplitude, right?  If not, we need to merge that PR first.

Because of this, I would multiply the desired p90 range here by sqrt expected power (to get a desired beta)

Reply:
Literally right here, on this line do the mult to keep it simple.  Also pass amp as second arg, obviously

Reply:
updated to use beta